InterviewFlashCards/IFCFlashCard.m

```diff
@@ -13,18 +12,16 @@ - (instancetype)initWithDictionary:(NSDictionary *)dict {
         self.answer           = dict[@"answer"];
         self.questionImageURL = dict[@"question_url"];
 
-        NSDictionary *urls    = dict[@"answer_urls"];
-        self.answerImageURLs = [self arrayOfUrlsFromDictionary:urls];
-
-        if (self.questionImageURL) {
-            self.questionImages = [NSMutableArray new];
-        }
-        if (self.answerImageURLs) {
-            self.answerImages = [NSMutableArray new];
+        id urls    = dict[@"answer_urls"];
+        if ([urls isKindOfClass:[NSArray class]]) {
+            self.answerImageURLs = urls;
+        } else if ([urls isKindOfClass:[NSDictionary class]]) {
```

Comment:
@vhart does this `else if` statement get executed ever? Based on the `algorithm-questions.json`above, it looks like all the values at `dict[@"answer_urls"]` are arrays, right? Or am I missing something?